### DISCUSSION TITLE
Sobre el formato de entrega

### DISCUSSION THREAD

**Discussion by mon-b** [IIC2133-PUC/2024-2#48] [Tarea 0] [answered]
Hola!

Me gustaría saber si hay que seguir algún formato de entrega en el repo, actualmente tengo subido todo lo que venía en el repo base y también los inputs (se me olvidó ponerlos en el gitignore). Debería borrar todo y dejar solo la carpeta src/puffle que tiene al main.c?

**Comment by sebacarrascop** [comment, marked as answer]
hola!, idealmente subirlo sin los test (se nos pasó añadirlo al gitignore del código base), pero no hay descuentos ni nada por eso.
